docs/installing.md

```diff
@@ -1,6 +1,161 @@
 Installing the JOSS application
 ===============================
 
-To be written...
+Any open journal (JOSS, JOSE, etc.) can be considered in three parts:
 
-For now, please take a look at the [JOSS codebase](https://github.com/openjournals/joss). 
+1. The website
+2. The Whedon gem
+3. A thin API wrapper around the Whedon gem to interface with GitHub
+
+For JOSS, these correspond to the:
+
+1. [JOSS](https://github.com/openjournals/joss),
+2. [Whedon](https://github.com/openjournals/whedon), and
+3. [Whedon-API](https://github.com/openjournals/whedon-api)
+
+code bases.
+
+Setting up a local development environment
+------------------------------------------
+
+All open journals are coded in Ruby,
+with the website and Whedon-API developed as
+[Ruby on Rails](https://rubyonrails.org/inst) applications.
+
+If you'd like to develop these locally,
+you'll need a working Ruby on Rails development environment.
+For more information, please see
+[this official guide](https://guides.rubyonrails.org/getting_started.html#creating-a-new-rails-project-installing-rails).
+
+Deploying your JOSS application
```

Comment:
Reading this section makes me wonder if we should add an `app.json` for this application, similar to what we have for Whedon-API: https://github.com/openjournals/whedon-api#deploying which would make it much easier to provision on Heroku.

Reply:
I think that would be great !

I will flag, though, that the `app.json` didn't catch a few things I ended up needing, like Cloudinary for the paper preview service !

Cloudinary is also now (newly?) prohibiting new free accounts from serving PDFs, so I had to reach out to their customer service to get it resolved. Definitely something I'll add in that section of the docs, but just to highlight how both docs and the `app.json` are helpful :smile_cat: